docs/ref/request-response.txt

```diff
@@ -1272,6 +1279,12 @@ with the following notable differences:
   :attr:`~StreamingHttpResponse.streaming_content` attribute. This can be used
   in middleware to wrap the response iterable, but should not be consumed.
 
+* It has no ``text`` attribute. This is because :class:`StreamingHttpResponse` is designed
+  for efficient memory usage by streaming content directly to the client. If you need
+  to access the response content as text, you can use the :attr:`~StreamingHttpResponse.getvalue` method to load
+  the entire content into memory and then convert it to text. However, be cautious as
+  this approach can lead to high memory usage for large content.
```

Comment:
I think we already describe why the content shouldn't be iterated/consumed 
```suggestion
* It has no ``text`` attribute, as it would require iterating the response object.
```

Reply:
Thank you for summarizing the key points so well.